processing/src/main/java/org/apache/druid/query/groupby/GroupByQueryHelper.java

```diff
@@ -42,6 +42,7 @@
 import org.apache.druid.query.aggregation.AggregatorFactory;
 import org.apache.druid.query.aggregation.PostAggregator;
 import org.apache.druid.query.dimension.DimensionSpec;
+import org.apache.druid.segment.RowBasedColumnSelectorFactory;
```

Comment:
It's needed now because it moved out of the `org.apache.druid.query.groupby` package. Anyway, we have a Travis step that verifies there aren't any unneeded imports, so there's no need to scan for them visually.